### ISSUE TITLE
Module fails on Kubuntu systems

### ISSUE THREAD

**Issue by Xliff** [tony-o/perl6-libyaml#4]
Module is failing its install for me. I am on the latest Kubuntu. Error message is as follows:

```
$ src/build.sh
gcc: error: libyaml_wrap.c: No such file or directory
gcc: fatal error: no input files
compilation terminated.
gcc: error: libyaml_wrap.o: No such file or directory
gcc: error: unrecognized command line option ‘-W1,-soname,libyaml_wrap.so.1’
If you're running locally you may need to add this dir to LD_LIBRARY_PATH
  to do that, run: export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/cbwood/Projects/unbitrot/perl6-libyaml"
```

The problem here is that the file to be compiled is called "yamlwrap.c", not "libyaml_wrap.c" as referenced in src/build.sh:

```
gcc -c -fPIC libyaml_wrap.c -o libyaml_wrap.o
```

Also, this line fails on Debian systems:

```
gcc -shared -W1,-soname,libyaml_wrap.so.1 -lyaml -o libyaml_wrap.so.1.0.0 libyaml_wrap.o
```

**Comment by Xliff** [comment]
This is an addendum to #3.

**Comment by tony-o** [comment]
@Xliff - this should be fixed with `0.0.6` or f4d432e

**Comment by tony-o** [comment]
@Xliff is this problem resolved?
